CHANGES.rst

```diff
@@ -61,6 +61,9 @@ astropy.table
   methods. With this ``**row`` becomes possible, as does, more simply, turning
   a ``Row`` into a dictionary with ``dict(row)``. [#9712]
 
+- Added a new features namely ``.items()`` and ``.values()`` which return as
```

Comment:
Change to:
```
- Added two new ``Table`` methods ``.items()`` and ``.values()``, which return
  respectively ``tbl.columns.items()`` (iterator over name, column tuples)  and 
  ``tbl.columns.values()`` (list of columns) for a ``Table`` object ``tbl``.
```
When you make this commit include `[docs only` in the commit message to prevent running all CI tests unnecessarily.

Reply:
thankyou @taldcroft for the suggestion.